analyzer.py

```diff
@@ -106,6 +107,14 @@ def main():
     filtered_jobs = filter_jobs(all_jobs)
     results = analyze_jobs(filtered_jobs, jenkins_server)
     report_to_graphite(args.statsd_host, args.statsd_port, args.graphite_key, results)
+
+    print('\n Full results (only jobs that were built and failed in last 24 hours):\n')
+    print_results(results)
+
+    filtered_jobs = filter_jobs(all_jobs, only_built_in_last_24_hours=False)
```

Comment:
assuming you implement changes from comment above, this can be also updated
btw. would be nice to have more detailed variables names, we have now 2 times filtered_jobs but this lists contains different jobs

Reply:
agree, and implemented.